### DISCUSSION TITLE
Could Outside collaborators show org icons?

### DISCUSSION THREAD

**Discussion by Kingwl** [community/community#22711] [New to GitHub]
<p>Seems Outside collaborators cannot been find in org members</p>
<p>And then i cannot show icons in my profile page</p>
<p>Thanks for your help</p>

**Comment by nadiajoyce** [comment]
<p>Hi <a class="mention" href="/kingwl">@kingwl</a>,</p>
<p>I want to make sure I’m understanding correctly. Are you hoping to have an organization avatar show up on your personal <a href="http://github.com" rel="nofollow noopener">github.com</a> profile as an outside collaborator? If so, that isn’t something that’s available at this time, but I can understand the value one might get in having that feature.</p>
<p>We’re always working to improve GitHub, and we consider every suggestion we receive. I’ve logged your feature request in our internal feature request list. Though I can’t guarantee anything or share a timeline for this, I can tell you that it’s been shared with the appropriate teams for consideration.</p>
<p>Please let me know if you have any other questions.</p>
<p>Cheers!</p>

**Comment by Kingwl** [comment]
<p>Thanks for the Response</p>
<p>Actualy, I means i wanna to show an org avatar that i was a outside collaborator, I cannot find myself in org members(naturally)，As the result, i cannot set the visibility of the org avatar.</p>
<p>I want to know if I am an outside collaborator, Could I set the org avatar to display to my profile page?</p>
<p>Look forward for your favourable response :xd</p>

**Comment by nadiajoyce** [comment, marked as answer]
<p>Hi again <a class="mention" href="/kingwl">@kingwl</a>,</p>
<p>Thank you for confirming. That’s what I thought you meant. Unfortunately, as mentioned previously, that isn’t something that’s available at this time, but I can understand the value one might get in having that feature. I did log that in our internal feature request list for future consideration, as promised.</p>
<p>Cheers!</p>

**Comment by Kingwl** [comment]
<p>Okay, Thank you very much, Have a good day😉</p>

**Comment by jbialy** [comment]
<p>Hi,</p>
<p>I just wanted to add that I also think this would be a nice feature to have. :grinning:Github gives lots of visibility, showing the org icon for outside contributors would be pretty cool!</p>

**Comment by bitmaks** [comment]
<p>I think this would be a nice way to let people show that they contribute to organisations that they might not have contributor access to. I have a few projects where I am an outside collaborator and it would be nice to be able to show it on my profile.</p>
<p>Really hope the team is able to implement this.</p>
